content/chapitres/tests.adoc

```diff
@@ -58,21 +58,28 @@ L'execution de tests nécessite un outillage non ajouté au projet
 
 == Ajout des Outils de Tests Unitaires au Projet (2/3)
 
-Ajoutez le bloc suivant au `pom.xml`
+* Ajoutez la dépendance suivante dans votre `pom.xml` (section `<dependencies>`):
++
+[source,xml]
+----
+<dependency>
+  <groupId>org.springframework.boot</groupId>
+  <artifactId>spring-boot-starter-test</artifactId>
+  <scope>test</scope>
+</dependency>
+----
 
+* Puis configurez le plugin Maven Spring Boot pour les tests unitaires (section `<build><plugins>`):
```

Comment:
As long as this works I'm fine with that change, this allows to split dependency declaration and plugin configuration.

Neat 👌

Reply:
I got this configuration from you (in https://github.com/cicd-lectures/menu-server/pull/7 😅)